src/DocumentFormat.OpenXml.Framework/OpenXmlElement.cs

```diff
@@ -796,11 +796,30 @@ public void RemoveNamespaceDeclaration(string prefix)
         /// Finds the first child element in type T.
         /// </summary>
         /// <typeparam name="T">Type of element.</typeparam>
-        /// <returns></returns>
+        /// <returns>The first child element of type T or null</returns>
         public T? GetFirstChild<T>()
             where T : OpenXmlElement
             => ChildElements.First<T>();
 
+        /// <summary>
+        /// Finds the first child element of type T or adds a new element if it does not exist.
```

Comment:
```suggestion
        /// Finds the first child element of <typeparam ref="T"/> or adds a new element if it does not exist.
```

or something like that so it references it in intellisense better

Reply:
updated